src/current/cockroachcloud/free-trial.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Try CockroachDB Cloud for free [WIP]
+summary: Learn about free trial credits for CockroachDB Cloud
+toc: true
+cloud: true
+--- 
+
+When you sign up for CockroachDB Cloud, your new organization's account starts with $400 in free credits, allowing you to try the product at no cost.
+
+## Free trial credits
+
+Free trial credit is applied to the first $400 in costs for all clusters created in your organization, regardless of the deployment type: CockroachDB Standard, Advanced, or Basic.
+
+Monthly billing begins when the free trial ends: on the day that the last credit is used, or upon expiration of credit, if any remains. By the end of the trial, you must provide a credit card for continued billing to prevent interruptions in service or loss of data.
```

Comment:
```suggestion
Monthly billing begins when the free trial ends: on the day that the last credit is used, or when any remaining credits expire. By the end of the trial, you must provide a payment method to prevent interruptions in service or loss of data.
```